### ISSUE TITLE
Implement the connection between the ordering system and btract

### ISSUE THREAD

**Issue by lukasmueller** [solgenomics/sgn#4033]
Expected Behavior 
--------------------------------------------------------------------------
btract needs additional information for providing status on orders, including an order number and status information.

For Bugs:
---------

**Comment by titima15** [comment]
Close this ticket because fieldbook will be used instead of ODK. A new ticket will be issued once template is ready.
